wranglertools/import_data.py

```diff
@@ -350,7 +404,7 @@ def get_sub_field(field_name):
     """Construct embeded field names."""
     try:
         return field_name.split(".")[1].rstrip('-0123456789')
-    except:  # pragma: no cover
+    except Exception:  # pragma: no cover
```

Comment:
Are these exceptions common? You might consider logging something in these cases if you hit one unintentionally?